ackhandler/sent_packet_handler.go

```diff
@@ -333,7 +333,11 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 			h.bytesInFlight,
 			h.congestion.GetCongestionWindow())
 	}
-	return !(congestionLimited || maxTrackedLimited)
+	// Workaround for #555:
+	// Always allow sending of retransmissions. This should probably be limited
+	// to RTOs, but we currently don't have a nice way of distinguishing them.
```

Comment:
Doesn't this completely defeat the purpose of congestion control? We detect congestion by observing loss events. If we now just blast out all retransmissions for lost packets, congestion control won't have any effect at all.